package.json

```diff
@@ -11,17 +11,17 @@
     "installDeps": "(cd src/zeeguu-react/ && npm install) && ln -s src/zeeguu-react/node_modules .",
     "start": "react-scripts start",
     "prebuild": "rimraf build",
-    "build": "cp ./public/manifest.chrome.json ./public/manifest.json && npm-run-all buildUnix:* && npm-run-all build:*",
-    "buildWindows": "copy .\\public\\manifest.chrome.json .\\public\\manifest.json && npm-run-all buildWindows:* && npm-run-all build:*",
+    "build": "npm-run-all buildUnix:* && npm-run-all build:* && cp ./public/manifest.chrome.json ./build/manifest.json",
+    "buildWindows": "copy .\\public\\manifest.chrome.json .\\build\\manifest.json && npm-run-all buildWindows:* && npm-run-all build:*",
     "buildUnix:popup": "INLINE_RUNTIME_CHUNK=false DISABLE_ESLINT_PLUGIN=true react-scripts build",
     "buildWindows:popup": "SET INLINE_RUNTIME_CHUNK=false && SET DISABLE_ESLINT_PLUGIN=true && react-scripts build",
     "build:content": "webpack --mode production ./src/JSInjection/main.js --output-path ./build/",
     "build:bg": "webpack --mode production ./src/background/background.js --output-path ./build/ --output-filename background.js",
     "build:izo": "webpack --mode production ./src/injectOnZeeguuOrg/injectOnZeeguuOrg.js --output-path ./build/ --output-filename injectOnZeeguuOrg.js",
     "pub": "zip -r build.zip build && open https://chrome.google.com/webstore/devconsole/7f6da896-c12c-4cb3-8ad7-b1641fc8efbc/ckncjmaednfephhbpeookmknhmjjodcd/edit/package",
-    "buildFirefox": "cp ./public/manifest.firefox.json ./public/manifest.json && npm-run-all build:*",
+    "buildFirefox": "npm-run-all build:* && cp ./public/manifest.firefox.json ./build/manifest.json",
     "pubFirefox": "(cd build && zip -r ../build.zip *) && open https://addons.mozilla.org/en-US/firefox/",
-    "pub-ci": "(cd src && git clone -b development https://github.com/zeeguu-ecosystem/zeeguu-react.git && cd zeeguu-react && npm ci) && cp ./public/manifest.chrome.json ./public/manifest.json && npm-run-all build:* && zip -r build.zip build",
+    "pub-ci": "(cd src && git clone -b development https://github.com/zeeguu-ecosystem/zeeguu-react.git && cd zeeguu-react && npm ci) && cp ./public/manifest.chrome.json ./build/manifest.json && npm-run-all build:* && zip -r build.zip build",
```

Comment:
These commands will also fail in windows, but since this is publishing maybe we should do it from a docker container that just compiles it and publishes it? This would also remove the need for all the different commands for Windows/Unix systems?

Reply:
that could be a good solution; less duplication, more docker :)

Reply:
do we go for this?

Reply:
Let's give it a try - I will see if I can create a docker deployment file to see if I can build it. Do you know if there is an easy way of getting files from the docker component out? I guess this would be needed to then get the packages.

Reply:
I guess we could essentially just make the container a map, and it just writes to the host machine. I will give it a try!

Reply:
@mircealungu  I have pushed the docker image and changed the readme with the instructions on how to use it. Let me know if my first try at building a docker container is OK. I can build both versions and have tested them and it works well.

Reply:
I've tested and it builds fine on my machine.

Reply:
I'll merge it now.